Dockerfile

```diff
@@ -1,16 +1,2 @@
-# [Build environment image]
-FROM golang:alpine AS builder
-
-WORKDIR $GOPATH
-RUN apk add --no-cache build-base wget
-RUN wget -O - -q https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | sh -s v1.30.0
-
-WORKDIR $GOPATH/src/github.com/exepirit/gitea-golangci-lint/
-COPY . .
-ENV GO111MODULE=on
-RUN go build -o $GOPATH/bin/gitea-golangci-lint ./...
-
-# [Final image]
-FROM golang:alpine
-COPY --from=builder /go/bin/golangci-lint /go/bin/golangci-lint
-COPY --from=builder /go/bin/gitea-golangci-lint /go/bin/gitea-integration
+FROM alpine:latest
+COPY ./gitea-golangci-lint /bin/gitea
```

Comment:
Is it better to collect whole application inside container instead of copy prebuilt binary file.

Reply:
Yes, if we use multistage dockerfile will be much more convenient. But the dockerfile will be more complicated, it's not necessary. Mostly, we just build this docker image one time. It's not updated so frequently. So just create a simple docker file.

Reply:
I understood meaning of this change :)
But GitHub workflow uses build inside container, so multistage image build need.